app/src/main/java/com/brainwallet/ui/BrainwalletActivity.kt

```diff
@@ -62,11 +63,11 @@ class BrainwalletActivity : BRActivity() {
         }
 
         setContent {
-            val appSetting by BrainwalletApp.module.settingRepository.settings.collectAsState(
+            val appSetting by BrainwalletApp.module!!.settingRepository.settings.collectAsState(
```

Comment:
🧑‍🎓 ✋ : What does this operator do?

Reply:
this is not-null assert: https://kotlinlang.org/docs/null-safety.html#not-null-assertion-operator